SampleARApp/SampleARApp/Utils/CoordinatesAdapter.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CoordinatesAdapter.swift
+//  SampleARApp
+//
+//  Created by Tony Ngo on 11.10.2022.
+//
+
+import UIKit
+
+struct CoordinatesAdapter {
+    var imageSize: CGSize = .zero
+    let screenSize: CGSize
+}
+
+extension CoordinatesAdapter {
+    func screenCoordinates(normalizedImagePoint: CGPoint) -> CGPoint {
```

Comment:
I'll add an explanation. As for the normalized coordinates, we want to pass them in because:
- Vision results are already normalized points in relation to the image provided to it.
- It saves us some operations to normalize actual coordinates.